src/Components/Facility/BedDeleteDialog.tsx

```diff
@@ -34,9 +18,14 @@ const BedDeleteDialog = (
     setDisable(true);
   };
   return (
-    <Dialog open={props.show} onClose={handleCancel}>
-      <DialogContent>
-        <DialogContentText
+    <DialogModal
+      show={props.show}
+      onClose={handleCancel}
+      title
```

Comment:
What's this line doing?

Reply:
show - It decides whether to show the dialog or not when clicked on delete bed.
onClose - It handles the cancel button/
title - Its a required prop and since the original dialog does not have any title, I passed it without one.

All the above three props are compulsory to pass since DialogModel expects it.

Reply:
Yes, let's make design consistent with other dialogs